src/alist_naudio.c

```diff
@@ -320,9 +330,8 @@ void alist_process_naudio_mp3(struct hle_t* hle)
 
 void alist_process_naudio_cbfd(struct hle_t* hle)
 {
-    /* TODO: see what differs from alist_process_naudio_mp3 */
     static const acmd_callback_t ABI[0x10] = {
-        UNKNOWN,        ADPCM,          CLEARBUFF,      ENVMIXER,
+        OVERLOAD,       ADPCM,          CLEARBUFF,      ENVMIXER,
```

Comment:
You might also want to add OVERLOAD to alist_process_naudio_mp3 as it also uses this very same audio command (no changes).

The only difference I could remember between mp3 and cbfd variants is in the MP3ADDY command, which is btw not implemented in rsp-hle and whose name is not really meaningful.

Still I would prefer if both variants are kept separated for documentation purpose.

Reply:
Thanks for the review! I've tested the unsigned version and it works as well (little surprising). I also added a comment containing the finding of the ucode differences.